doc/source/changes/version_0_26.rst.inc

```diff
@@ -306,21 +306,14 @@ Miscellaneous improvements
      b [2]: 'b0' 'b1'
     dtype: float64
 
-* To create a 1D array using from_string() or read_csv() and the default separator ' ', a `nan`, `NaN` or a tabulation
-  character `\t` must be used in front of the data line:
+* To create a 1D array using from_string() and the default separator ' ', a tabulation character `\t` must be added
+  in front of the data line:
```

Comment:
s/a tabulation character `\t`/a tabulation character `\t` (instead of `-` previously)/